buildSrc/src/main/groovy/library.deps-conventions.gradle

```diff
@@ -3,11 +3,19 @@
 
 // Define a standard set of repositories, a specific library may need to customize this list and add or remove some items
 repositories {
+    flatDir dirs: "${rootProject.projectDir}/libs/x64"
```

Comment:
It is probably a good idea to condense all the (non-eclipse) jars into a single place to avoid them being scattered around and introducing a higher chance for duplicates.

If there are platform-specific components (eg, DLLs) it's probably best to separate them and label them with a common platform name, such as "win-x86_64" rather than just "x64".

Reply:
third-party jars have been refactored directly in to the /libs folder as suggested. Eclipse jars have been moved into a root level folder called /eclipse-luna-libs. We were unable to find a solution to pull these libs from any repository and resolved to include them in them in the project repo until an alternative solution is identified and/or the eclipse version is updated to one that is available and retrievable. 

DLLs have been moved into /libs/win-x64 directory as suggested.

Reply:
It was more efficient to include all the eclipse jars than to chase down just the ones required to compile. 
It is planned to extract them into a single zip in the github packages once the installer is building.

Reply:
Put the issue that's tracking the zip replacement as a comment in the build file.

Reply:
Added issue link to the build file. https://github.com/CentralValleyModeling/wrims/issues/148